04.ls/ls_command.rb

```diff
@@ -5,47 +5,50 @@
 COLUMN_COUNT = 3
 
 def main
-  files = if a_option?
-            fetch_files(option: 'a')
-          else
-            fetch_files
-          end
-  puts format_to_show(files)
+  options = load_options
+  sorted_files = fetch_files.sort
+
+  filtered_files = if options.key?(:a)
+                     sorted_files
+                   else
+                     sorted_files.reject { |file| file.start_with?('.') }
```

Comment:
ちょっと高等テクニックですが、こんな書き方もできます。

```suggestion
                     sorted_files.grep_v(/^\./)
```

その上で三項演算子を使ってif文を置き換えるとコードがスッキリするかも？